extended/src/main/java/io/kubernetes/client/extended/kubectl/KubectlLog.java

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.extended.kubectl;
+
+import io.kubernetes.client.PodLogs;
+import io.kubernetes.client.extended.kubectl.exception.KubectlException;
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import io.kubernetes.client.openapi.models.V1Pod;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class KubectlLog extends Kubectl.ResourceAndContainerBuilder<V1Pod, KubectlLog>
+    implements Kubectl.Executable<V1Pod> {
```

Comment:
`Kubectl.Executable<InputStream>` will make it returning an input stream so `stream()` can be removed

Reply:
done.